lib/ExtUtils/MakeMaker.pm

```diff
@@ -427,6 +427,22 @@ sub _has_cpan_meta_requirements {
     };
 }
 
+sub _min_EUMM {
+    my ($self) = @_;
+    my @eumm_versions = sort grep defined, map {
+        $_->{'ExtUtils::MakeMaker'}
+    } grep defined, @{$self}{@PREREQ_KEYS};
```

Comment:
CONFIGURE_REQUIRES is the only place that is useful for the place this is being used, so checking all of the phases is incorrect.  It's also possible to provide the configure requires only via META_ADD/META_MERGE, so this could result in spurious warnings.